### DISCUSSION TITLE
Make sure you put this plugin after sort-imports

### DISCUSSION THREAD

**Discussion by vincerubinetti** [hosseinmd/prettier-plugin-jsdoc#200] [Show and tell]
I'm using the following Prettier plugins:

```json
  "plugins": [
    "@ianvs/prettier-plugin-sort-imports",
    "./node_modules/prettier-plugin-jsdoc/dist/index.js",
    "prettier-plugin-css-order"
  ]
```

Using Prettier v3 and v1.0.1 of this plugin. For some reason it doesn't work (vscode doesn't show warning underline and command line detects no problems) if this plugin comes before `/prettier-plugin-sort-imports`. I know plugin order matters, but I didn't think these two plugins would be touching the same parts of the code and conflicting with each other.

**Comment by fardad-dev** [comment]
Sort-import and jsdoc both manipulating the main plugin, so to do that we should keep the previous manipulation, so we did it in jsdoc.

**Reply by vincerubinetti** [reply]
Hmm interesting, so presumably the sort-imports plugin doesn't preserve the previous manipulation, and your plugin does? Maybe I should create an issue on their repo. 

Edit: they have one already I think https://github.com/IanVS/prettier-plugin-sort-imports/issues/27
